Cargo.toml

```diff
@@ -11,20 +11,24 @@ publish = true
 [features]
 abi = ["borsh/unstable__schema", "schemars-stable"]
 abi-alpha = ["borsh/unstable__schema", "schemars-alpha"]
+abi-with-schemars-v1 = ["borsh/unstable__schema", "schemars-v1"]
+abi-with-schemars-v0_8 = ["borsh/unstable__schema", "schemars-v0_8"]
 # This feature was introduced for legacy reasons for nearcore, and MUST be avoided
 # https://github.com/near/nearcore/pull/4621#issuecomment-892099860
 internal_unstable = []
 default = []
 schemars = []
-schemars-stable = ["schemars", "dep:schemars-stable"]
-schemars-alpha = ["schemars", "dep:schemars-alpha"]
+schemars-v1 = ["schemars", "dep:schemars-v1"]
+schemars-v0_8 = ["schemars", "dep:schemars-v0_8"] 
+schemars-alpha = ["schemars", "dep:schemars-v1"]
+schemars-stable = ["schemars", "dep:schemars-v0_8"]
```

Comment:
`# Deprecated` comments have been added to `[dependencies]` instead of `[features]`, 
and then removed in 3rd commit